README.md

```diff
@@ -99,6 +99,9 @@ Bash script dependencies
 
 Production Setup
 =====
+
```

Comment:
We should document this, but I'm not sure this README is necessarily the right place or that we need to do it right now. Thoughts?

Reply:
A README inside `ht_tanka/environments/metadata_processing/post_zephir` would work for me.

Reply:
:+1: I'll remove this section here, since it is no longer relevant.